Polyglot.safariextension/keymap.js

```diff
@@ -0,0 +1,55 @@
+const keymap = {
+  ' ': 'Space',
+  '0': 'Digit0',
+  '1': 'Digit1',
+  '2': 'Digit2',
+  '3': 'Digit3',
+  '4': 'Digit4',
+  '5': 'Digit5',
+  '6': 'Digit6',
+  '7': 'Digit7',
+  '8': 'Digit8',
+  '9': 'Digit9',
+  'a': 'KeyA',
+  'b': 'KeyB',
+  'c': 'KeyC',
+  'd': 'KeyD',
+  'e': 'KeyE',
+  'f': 'KeyF',
+  'g': 'KeyG',
+  'h': 'KeyH',
+  'i': 'KeyI',
+  'j': 'KeyJ',
+  'k': 'KeyK',
+  'l': 'KeyL',
+  'm': 'KeyM',
+  'n': 'KeyN',
+  'o': 'KeyO',
+  'p': 'KeyP',
+  'q': 'KeyQ',
+  'r': 'KeyR',
+  's': 'KeyS',
+  't': 'KeyT',
+  'u': 'KeyU',
+  'v': 'KeyV',
+  'w': 'KeyW',
+  'x': 'KeyX',
+  'y': 'KeyY',
+  'z': 'KeyZ',
+  ';': 'Semicolon',
+  '=': 'Equal',
+  ',': 'Comma',
+  '-': 'Minus',
+  '.': 'Period',
+  '/': 'Slash',
+  '`': 'Backquote',
+  '[': 'BracketLeft',
+  '\\': 'Backslash',
+  ']': 'BracketRight',
+  '\'': 'Quote',
+  '§': 'IntlBackslash',
+}
```

Comment:
Just to be sure: Is this keymaps covering all possible key events?
I'm curious what happens when this function was passed with a key event for `¥` in JIS keyboard.

Reply:
@uetchy you're right. It won't work if a user configured shortcut in a different keyboard layout than EN. Please can you tell me what the `event.code` for the `¥` according to http://keycode.info? I found that it has `charCode=165`. I think how to get EN character from it. Or maybe just as an idea, add some restrictions that a user can add only in EN 😀 ?